common/configstore/configstore.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package configstore
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric-protos-go-apiv2/common"
+	"github.com/hyperledger/fabric-x-orderer/common/filerepo"
+	"github.com/pkg/errors"
+)
+
+const (
+	fileSuffix        = "pb.bin"
+	fileNameSeparator = "-"
+	fileNameTemplate  = "configblock" + fileNameSeparator + "%d"
+)
+
+// Store holds a set of numbered blocks, typically config blocks.
+// Blocks get be added, listed, and retrieved.
+// A convenience method allows fetching the last block.
+type Store struct {
+	fileRepo *filerepo.Repo
+}
+
+// NewStore initializes a new config store at the storeDir.
+// All file system operations on the returned file repo are thread safe.
+func NewStore(storeDir string) (*Store, error) {
+	repo, err := filerepo.New(storeDir, fileSuffix)
```

Comment:
why use file suffix here?

Reply:
The filerepo lib wants a file suffix, as it can handle multiple types of files by their suffix, in the same folder.